ast/field.go

```diff
@@ -50,7 +50,7 @@ var _ FieldDeclNode = NoSourceNode{}
 //	optional string foo = 1;
 type FieldNode struct {
```

Comment:
I dont think changes here are backwards-incompatible since I only added some exported nodes and didn't change any exported APIs (just to make sure we are looking at the same commit).